fruit/fruit.js

```diff
@@ -0,0 +1,19 @@
+
+class Fruit {
+    constructor (quality) {
+        this.quality = quality
+    }
+
+    produce() {
+        let roll = Math.ceil(Math.random()*10)
+        if(roll > 5) {
+            this.quality = 'good'
+            return this
+        } else {
+            this.quality = 'bad'
+            return this
+        }
+    }
+}
```

Comment:
langsung aja diconstructor nya this.quality = Math.ceil(Math.random()*10) > 5 ? 'good' : 'bad'